### PR TITLE
Clean up repo kind of?

### PR DESCRIPTION
**Pull request by h4rldev**

### CODE REVIEW

File: homme/main.py
Comment (charlie-sans) [inline homme/main.py:1]: uhh what happened here? it's all just red

File: addons/Notepad.py
Comment (charlie-sans) [inline addons/Notepad.py:1]: if i include files that i got from github that i moved from one folder to another without the required things, i like to credit the creator in the file itself!

File: addons/tkinter-calc.py
Comment (charlie-sans) [inline addons/tkinter-calc.py:1]: must have run into a error when i got the file? i have the repo on another pc and ill update the file when i get the origonal repo!

File: home/midiplayer.py
Comment (charlie-sans) [inline home/midiplayer.py:1]: when you run it, the output will be hidden unless you open the output which aint here yet.

File: python/bluescreen.py
Comment (charlie-sans) [inline python/bluescreen.py:1]: i must have forgotten to remove the lines before i pushed the file! my mestake

File: python/file.py
Comment (charlie-sans) [inline python/file.py:1]: well i plan for python to do the desktop widgets type thing and everything else like interacting with the hardware is in c, c++ or rust!

File: python/fileloop.py
Comment (charlie-sans) [inline python/fileloop.py:1]: that was a duped file that not even i know what it does, ill have a look at it and probs del it.

File: python/loginscreen.py
Comment (charlie-sans) [inline python/loginscreen.py:1]: file is a .PY script that hosts the locations of system folders that other scripts can use to work with without having to define it in the file as if they did that, any folder structure update would probably break there code.

File: python/main.py
Comment (charlie-sans) [inline python/main.py:1]: cheers on this one!

File: python/registry.py
Comment (charlie-sans) [inline python/registry.py:1]: i'm still trying to figure out what is a good option, i have json in mind that acts like INI, and so ill see how that works.

File: python/setup.py
Comment (charlie-sans) [inline python/setup.py:1]: when i created setup.py, this is the file that got duped into fileloop.py. i must have forgotten to delete them. i plan to bake them into the main.py script.

File: python/shell.py
Comment (charlie-sans) [inline python/shell.py:1]: cheers on this one!

File: python/soundfonts.py
Comment (charlie-sans) [inline python/soundfonts.py:1]: fortunately not for some reason?

File: python/towii.py
Comment (charlie-sans) [inline python/towii.py:1]: nope! i tried to make it work. failed HARD. instead i will have them talk somehow

**Review by charlie-sans — APPROVED** [review]
all looks good!

File: homme/main.py
Comment (h4rldev) [reply homme/main.py:1]: It's just a duplicate of a file so i thought it was safe to remove

**Pull request merged**